src/components/components/Countup.res

```diff
@@ -27,31 +31,48 @@ module Basic = {
     let startCountup = countUpObj->start
     let updateCountup = countUpObj->update
     let (showCountupStyles, setShowCountupStyles) = React.useState(_ => false)
+    // NOTE: this is a little hack to make sure the countup doesn't show incorrect intermediate values. Eg if it 'ticks over' from 99->1 (or 01) it shouldn't count down.
+    let ((changeInDigitLength, pevPrefixLength), setChangeInDigitLength) = React.useState(_ => (
+      false,
+      0,
+    ))
```

Comment:
This took me ages to work out, but now the counter never does weird things like count backwards